public/index.php

```diff
@@ -1,5 +1,10 @@
 <?php
 
+$filename = __DIR__.preg_replace('#(\?.*)$#', '', isset($_SERVER['REQUEST_URI']) ? $_SERVER['REQUEST_URI'] : '');
```

Comment:
I am unfamiliar with Laravel and prefer to do local development versus Homestead/similar solutions.